olp-cpp-sdk-authentication/src/TokenProviderImplPrivate.cpp

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
I guess we can put this also in TokenProvider.cpp as we reached our goal of exposing internals trough the header.

Reply:
Moved TokenProviderImplPrivate to TokenProvider.cpp